docs/source/configuration/index.rst

```diff
@@ -14,3 +14,102 @@ The AtlasDB configuration has two main blocks - keyValueService and leader. Plea
    cassandra_KVS_configuration
    leader_config
    logging
+
+There are three valid configuration modes, listed below.
+
+Leader block
+------------
+
+This configuration setup is required to use Cassandra KVS with more than one node.
+
+A minimal AtlasDB configuration for running against cassandra will look like the below.
+
+Importantly - your lock creator must be consistent across all nodes. If you do not provide a lock creator, it will default to the first host
```

Comment:
It isn't the first host in the leaders list. It is sorted. Also, this seems like partial duplication of `docs/source/configuration/leader_config.rst`